docs/guides/audio-config.md

```diff
@@ -20,33 +22,18 @@ If not present, you'll have to update your bootup kernel params:
 
 # Audio Configuration Files
 
-In most scenarios, you should use [these files](https://gist.github.com/MCMrARM/c357291e4e5c18894bea10665dcebffb), following the instructions in that gist's `README.md`.
-
-Special scenarios are:
-
-- [2019 16" MacBook Pro audio files](https://gist.github.com/kevineinarsson/8e5e92664f97508277fefef1b8015fba) - that laptop has 6 speakers and needs slightly different config
-- [2020 13" MacBook Air audio files](https://gist.github.com/bigbadmonster17/8b670ae29e0b7be2b73887f3f37a057b)
-
-# Using Pipewire instead of Pulseaudio
-
-You'll need to modify the `/lib/udev/rules.d/91-pulseaudio-custom.rules` file from the links above.
-
-```diff
-SUBSYSTEM!="sound", GOTO="pulseaudio_end"
-ACTION!="change", GOTO="pulseaudio_end"
-KERNEL!="card*", GOTO="pulseaudio_end"
-
--SUBSYSTEMS=="pci", ATTRS{vendor}=="0x106b", ATTRS{device}=="0x1803", ENV{PULSE_PROFILE_SET}="apple-t2.conf"
-+SUBSYSTEMS=="pci", ATTRS{vendor}=="0x106b", ATTRS{device}=="0x1803", ENV{PULSE_PROFILE_SET}="apple-t2.conf", ENV{ACP_PROFILE_SET}="apple-t2.conf"
+Simply run the following to set up audio :-
 
-LABEL="pulseaudio_end"
+```bash
+git clone https://github.com/kekrby/t2-better-audio.git
+cd t2-better-audio
+./install.sh
+sudo rm -r $HOME/t2-better-audio
```

Comment:
it was cloned without sudo so why is sudo needed to remove it? also this assumes the user was in their home directory when they cloned the repo

Reply:
There is a strange issue on Ubuntu that causes error stating something like removing read only files if removing something from $HOME directory.

Reply:
hmm maybe it'd be simpler to clone it into /tmp? or use rm -rf?

Reply:
411234d872de947f2792fa3d23e0d26811fdc8eb

Also, I was thinking to attach the links of the apt repo as an easy alternative method.

Reply:
You probably could also add the link for your Pacman repo?

Reply:
I think it should be mentioned that this guide should be followed only if the distro being used does not package the files as installing them as a distro package would allow updates to happen just like any other package but manual installation would not.

Reply:
I guess we can make this change in a separate PR since I've to work on the apt repo a bit more.

Reply:
I'll merge this if it's good apart from this.

Any other suggestions @Redecorating @kekrby?